server/database/queries/getNews.js

```diff
@@ -0,0 +1,8 @@
+const connection = require('../config/connection');
+
+const getNews = () => {
+  const sql = 'SELECT * FROM news';
```

Comment:
can you add order by to the query , it's will be good to return the news order by descending data